packages/arb-validator-core/message/data.go

```diff
@@ -104,11 +118,8 @@ func ByteStackToHex(val value.Value) ([]byte, error) {
 	return buf.Bytes()[:intLength], nil
 }
 
-func BytesToByteStack(val []byte) value.Value {
-	chunks := bytesToIntValues(val)
-	ret := value.NewEmptyTuple()
-	for _, chunk := range chunks {
-		ret = value.NewTuple2(ret, value.NewIntValue(chunk))
-	}
+func BytesToByteStack(val []byte) value.TupleValue {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function BytesToByteStack should have comment or be unexported](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)